support-frontend/assets/components/directDebit/directDebitActions.ts

```diff
@@ -196,6 +189,13 @@ function confirmDirectDebitClicked(
 	onPaymentAuthorisation: (authorisation: PaymentAuthorisation) => void,
 ): (dispatch: Dispatch<Action>, getState: () => CheckoutState) => void {
 	return (dispatch, getState) => {
+		const recaptchaCompleted = getState().page.checkoutForm.recaptcha.completed;
+		if (!recaptchaCompleted) {
+			return dispatch(
+				setDirectDebitFormError("Please check the 'I'm not a robot' checkbox"),
+			);
+		}
+
```

Comment:
we moved the recaptcha check from pay callback (stage 1) to the confirm callback (stage 2)